projects/storefrontlib/src/lib/my-account/order/order-history/order-history.component.ts

```diff
@@ -62,7 +63,7 @@ export class OrderHistoryComponent implements OnInit, OnDestroy {
     }
   }
 
-  changeSortCode(sortCode: string) {
+  changeSortCode(sortCode: string): void {
     const event = {
```

Comment:
Can we add a type here?

Reply:
Don't see any models with a key of `sortCode`

Reply:
You have an option to maybe type it like `{ sortCode: string; currentPage: number }`. It's the same type you used in `fetchOrders()` on line 86